.codeclimate.yml

```diff
@@ -1,57 +1,6 @@
 ---
-engines:
-  csslint:
-    enabled: true
-  duplication:
-    enabled: true
-    config:
-      languages:
-      - javascript
-      - php
-  fixme:
-    enabled: true
-  phpcodesniffer:
-    enabled: true
-    config:
-      standard: "WordPress"
-  phpmd:
-    enabled: true
-  eslint:
-    enabled: true
-    channel: "eslint-2"
-  scss-lint:
-    enabled: true
-  markdownlint:
-    enabled: true
-ratings:
-  paths:
-  - "**.css"
-  - "**.scss"
-  - "**.inc"
-  - "**.js"
-  - "**.jsx"
-  - "**.php"
-  - "**.md"
-exclude_paths:
-  - "**/.github/**"
-  - "**/tests/**"
-  - "**.png"
-  - "**.jpg"
-  - "**.gif"
-  - "gulpfile.js"
-  - "composer.lock"
-  - ".phpcs.xml.dist"
-  - "**.json"
-  - "**.jshintrc"
-  - "**.jshintignore"
-  - "**.svnignore"
-  - "**.travis.yml"
-  - "**.gitignore"
-  - "**.eslintrc.js"
-  - "**.editorconfig"
-  - "**.pot"
-  - "**.txt"
-  - "**-min.js"
-  - "**-min.css"
-  - "**.dist"
-  - "**.sh"
+prepare:
+  fetch:
+    - url: "https://raw.githubusercontent.com/automattic/jetpack/master/.github/codeclimate.yml"
+      path: "codeclimate.yml"
```

Comment:
I'm not sure how this works, but shouldn't this be `codeclimate-base.yml`?

Reply:
🤦‍♂️ Yes you're right. I can't even follow my own documentation from the other file.

Updated in e8deacccef175b5c2f9d785a2327f5d6baeb9635